Cargo.toml

```diff
@@ -89,3 +85,7 @@ u128 = ["byteorder"]
 const_fn = ["nightly"]
 # Nightly marker feature gate
 nightly = []
+
+[target.'cfg(all(windows, target_env="msrv"))'.dependencies.winapi]
```

Comment:
Should this be `msvc`?

Reply:
Also, do we even need the `target_env` here? If you're building with a GNU toolchain on Windows you might still want `winapi`, right?

Reply:
> Should this be `msvc`?

oops



> Also, do we even need the `target_env` here? If you're building with a GNU toolchain on Windows you might still want `winapi`, right?

true. will adjust this when I can